gradle/build.go

```diff
@@ -83,7 +83,7 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 	dc.Logger = b.Logger
 
 	command := filepath.Join(context.Application.Path, "gradlew")
-	if _, err := os.Stat(command); os.IsNotExist(err) {
+	if fi, err := os.Stat(command); os.IsNotExist(err) {
```

Comment:
```suggestion
	if _, err := os.Stat(command); os.IsNotExist(err) {
```

Sorry, this needs to be `_` as well since we're not using `fi` any more.